catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/QueryOperations.java

```diff
@@ -657,4 +571,126 @@ private Filter getTagsQueryFilter() {
                                 .attribute(Metacard.TAGS)
                                 .empty());
     }
+
+    static class QuerySources {
```

Comment:
It has no usages outside of `QueryOperations` and its test class so I'm not sure I see the value.